style.css

```
@@ -0,0 +1,150 @@
+* {
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+  color: #fff;
+  font-size: 12px;
+  cursor: pointer;
+}
+
+li {
+  list-style: none;
+}
+
+body {
+  line-height: 1.4;
+  font-size: 16px;
+  font-family: 'Roboto', sans-serif;
+  color: grey;
+}
+
+/* main */
+
+main {
+  width: 100%;
+}
+
+header {
+  background: url(images/your_verse_hero.jpg) 50% 100% no-repeat;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 535px;
+  margin: 0 auto;
+  max-width: 100%;
+}
```

max-width should be a pixel size, and if you want percentages you should put it in a with property, this way you have your responsive width with the boundaries that you want.
```suggestion
header {
  background: url(images/your_verse_hero.jpg) 50% 100% no-repeat;
  display: flex;
  flex-direction: column;
  align-items: center;
  height: 535px;
  margin: 0 auto;
  max-width: 1440px;
  width: 100%;
}
```